docs/api-overview.md

```diff
@@ -2705,3 +2720,16 @@ _Appears in:_
 | `url` _string_ |  |  |  |
 
 
+#### Traces
+
+
+
+
+
+
+
+_Appears in:_
+- [MonitoringSpec](#monitoringspec)
```

Comment:
Same removal here I think.

Reply:
I've ran the make generate api-docs command but that line still remains, should i remove it manually?

Reply:
I think due to the common spec and the monitoring spec being linked it will result in them being shown up in the monitoring spec